packages/terra-form-select/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Updated `select-wrapper` styles to consume 100% of parent div's width.
```

Comment:
did you mean `search-wrapper` instead of `select-wrapper`? Could make comment a little more general:
```suggestion
  * Updated the specificity conflict for combobox's input flex & width styles
```